src/_pytest/config/findpaths.py

```diff
@@ -18,7 +21,13 @@ def exists(path, ignore=EnvironmentError):
         return False
 
 
-def getcfg(args, config=None):
+def getcfg(
+    args: List[py.path.local], config: Optional[Config] = None
```

Comment:
The `List` can be `Iterable[py.path.local]`.

You will have to rename the internal `args`, that's a bit annoying.

Reply:
> The `List` can be `Iterable[py.path.local]`.

Thanks.

> You will have to rename the internal `args`, that's a bit annoying.

No?  (You've meant the first `args = [x for x in args if not str(x).startswith("-")]`)